plugins/module_utils/redfish_utils.py

```diff
@@ -63,11 +69,23 @@ def get_request(self, uri):
         req_headers = dict(GET_HEADERS)
         username, password, basic_auth = self._auth_params(req_headers)
         try:
-            resp = open_url(uri, method="GET", headers=req_headers,
-                            url_username=username, url_password=password,
-                            force_basic_auth=basic_auth, validate_certs=False,
+            #To support HPE operations
+            if os.path.isfile(home + "/sessionfile.txt"):
```

Comment:
The correct way to concatenate paths is using `os.path.join`.

Reply:
The reason to include os.path.isfile here instead of os.path.join is to check if the sessionfile exists. This is to recognise that session id needs to be used instead of performing manual login.